ledger/participant-integration-api/src/main/scala/platform/store/backend/DBDTOV1.scala

```diff
@@ -5,11 +5,16 @@ package com.daml.platform.store.backend
 
 import java.time.Instant
 
-trait DBDTOV1
+import com.daml.scalautil.NeverEqualsOverride
+
+sealed trait DBDTOV1
```

Comment:
What does the V1 at the end signify? Can we drop it?

Reply:
yes, and also we will, FYI: not part of this review